frontend/discourse/app/components/reviewable-refresh/ip-lookup.gjs

```diff
@@ -0,0 +1,30 @@
+import Component from "@glimmer/component";
+import { service } from "@ember/service";
+import { optionalRequire } from "discourse/lib/utilities";
+
+export default class ReviewableIpLookup extends Component {
+  @service currentUser;
+
+  get showIpLookup() {
+    return (
+      this.args.reviewable.type !== "ReviewableUser" &&
```

Comment:
Should a post from a new user also show this? I'm assuming that might be ReviewablePost